spec/models/rating_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Rating, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Adding form validations will require setting some parameters in Rating model. Ensure to have validation tests here.